kubeassert/.kubeassert/hbase-table-row-count.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+##
+# @Name: hbase-table-row-count
+# @Description: Assert specified table has been has been created
+# @Usage: kubectl assert hbase-table-row-count
+##
+
+function hbase-table-row-count {
+    NAMESPACE=${1}
+    logger::info "NAMESPACE: $NAMESPACE"
+
+    # Validate input arguments
+    [[ -z $2 ]] && logger::error "You must specify a count of expected rows" && exit 1
+
+    export HBASE_INTERACTIVE_POD=$(kubectl -n ${NAMESPACE} get pods -o=name | grep hbase-interactive | sed "s/^.\{4\}//")
+    logger::info "HBASE_INTERACTIVE_POD: $HBASE_INTERACTIVE_POD"
+
+    logger::assert "Table should contain $2 rows."
+    kubectl -n ${NAMESPACE} exec -t ${HBASE_INTERACTIVE_POD} -- /bin/bash -c "echo \"count 'wines'\" | /stackable/hbase/bin/hbase shell -n"
```

Comment:
```suggestion
    kubectl -n ${NAMESPACE} exec -t hbase-interactive -- /bin/bash -c "echo \"count 'wines'\" | /stackable/hbase/bin/hbase shell -n"
```
(there appears to be a problem with kubectl-assert + chained commands in some environments, but as the pod name is fixed it can be used directly here)